packages/docusaurus-plugin-content-blog/src/plugin-content-blog.d.ts

```diff
@@ -190,6 +195,8 @@ declare module '@docusaurus/plugin-content-blog' {
     readonly formattedDate: string;
     /** Full link including base URL. */
     readonly permalink: string;
+    /** Alias full links including base URL. */
+    readonly aliases: string[] | undefined;
```

Comment:
was wondering maybe we should always have an array here? It's not very expensive and if you look at tags we already do this (tags can't be undefined in metadata, only frontMatter)

Reply:
Yeah - totally could